### DISCUSSION TITLE
openDTU on Battery: "Inverter Gesamtertrag Heute"

### DISCUSSION THREAD

**Discussion by Fokker-1** [tbnobody/OpenDTU#2390] [Ideas]
Hallo,

bei der openDTU on Battery gibt es den Eintrag "Inverter Gesamtertrag Heute".
Wenn ich das richtig sehe, ist die Zeitangabe **"Heute"** nicht passend zu dem dazu angegebenen Wert in Wh.

Dieser scheint kontinuierlich - auch über die Tagesgrenzen hinweg - aufaddiert zu werden, oder sehe ich das falsch?

Damit wäre das Verhalten grundsätzlich etwa so, wie auch bei "Inverter Gesamtertag Insgesamt"?

Danke im Voraus für eine Info.

Grüße
Werner

**Comment by schlimmchen** [comment]
Naja, das musst du dann bitte im Downstream-Projekt besprechen, das hat bei OpenDTU nichts zu suchen. Der Grund ich sicherlich jener, dass dein Inverter an einer Batterie hängt und du versäumt hast per Einstellung oder externen Mitteln den Wechselrichter täglich einmal neu zu starten. Die Wechselrichter haben kein Verständnis von Tagen. Der Ertrag von "heute" setzt sich beim Betrieb wie er vom Hersteller vorgesehen jede Nacht von selbst zurück, weil kein Licht da ist und der WR beim ersten Licht neu aufwacht. Wenn der WR an einre Batterie hängt, dann musst du selbst für solch einen Neustart sorgen.

**Reply by Fokker-1** [reply]
Hallo,

vielen Dank für die schnellen Erläuterungen, prima.

Es ist für den Laien immer etwas schwierig, die Hintergründe zu bewerten.
Aber ok., jetzt ist das klar, dann werde ich in der DTU mal ein zeitgesteuertes Reset eingeben und weitersehen.

Es ist auch immer etwas unklar für mich, wo ich hier in welchem Bereich welchen Inhalt einstelle.

Viele Grüße
Werner

**Reply by schlimmchen** [reply]
Wenn du OpenDTU-OnBattery einsetzt, dann melde ich pauschal immer erst dort: https://github.com/hoylabs/OpenDTU-OnBattery So steht es auch im "About" ("Über") Teil der Web UI :wink:

Du suchst vermutlich den [automatischen Neustart](https://opendtu-onbattery.net/firmware/configuration/dpl/#automatic-restart-time), weil du vermutlich den DPL benutzt.

Wenn du weitere Fragen hast, starte gerne eine [neue Diskussion](https://github.com/hoylabs/OpenDTU-OnBattery/discussions).

**Reply by Fokker-1** [reply]
Hallo,

vielen Dank für die Hilfe, dann weiß ich jetzt schon mal, wo ich zu welchem Aspekt hingehen muss.

Viele Grüße
Werner
